src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3299,7 +3299,7 @@ A catch() block after a catch (System.Exception e) block can catch non-CLS excep
     <value>Could not find an implementation of the query pattern for source type '{0}'.  '{1}' not found.  Consider explicitly specifying the type of the range variable '{2}'.</value>
   </data>
   <data name="ERR_QueryNoProviderStandard" xml:space="preserve">
-    <value>Could not find an implementation of the query pattern for source type '{0}'.  '{1}' not found.  Are you missing a reference to 'System.Core.dll' or a using directive for 'System.Linq'?</value>
```

Comment:
>reference to 'System.Core.dll' or a  [](start = 125, length = 36)

Should we still mention references, but without any specific names? Something like: "Are you missing references to required assemblies or a using directive for 'System.Linq'?"

Reply:
That's a fair point. 

I don't see any issues adding that information to the message. mentioning the reference assemblies but not specific names might make it more clear then just mentioning the using. I can make that change here in a bit.

Reply:
Got a bit sidetracked with work, but I just added this commit. 
https://github.com/dotnet/roslyn/pull/45920/commits/7a96e074a1f4159054c7052ae7d8da082b83e3f5

it changes the message to  "Are you missing required assembly references or a using directive for 'System.Linq'?"